Source/Common/Config.cpp

```diff
@@ -188,6 +190,12 @@ namespace JSON {
     , Config{std::move(ConfigFile)} {
   }
 
+
+  MainLoader::MainLoader(FEXCore::Config::LayerType Type, const char* ConfigFile)
+    : OptionMapper(Type)
+    , Config{std::move(ConfigFile)} {
```

Comment:
Noticed just after approving, but it doesn't seem like std::move would do anything here.

Reply:
Dropped the move